config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -1,3 +1,22 @@
+presets:
+- labels:
+    preset-kind-at-prow-enabled: "true"
```

Comment:
nit: let's make this `preset-kind-volume-mounts: "true"` since it just does the volume mounts.

Reply:
done, thx